### ISSUE TITLE
What is the namespace for RouterDB?

### ISSUE THREAD

**Issue by pajaro5** [OsmSharp/routing#7]
Hello
I'm trying to run the CalculateMatrix sample from OsmSharp with my own data.  I understand that first its necessary to create a router db instance.

```
// using OsmSharp.Routing;
// using OsmSharp.Routing.Osm;
// using OsmSharp.Routing.Osm.Vehicles;

var routerDb = new RouterDb();
```
But it doesn't works because the IDE can not find the namespace for the routerDb object.
Which one is the correct namespace? Or what should I do in order to create a serialized router db file?

**Comment by alecava58** [comment]
Hi, the namespace for RouterDb is OsmSharp.Routing

The following is the snippet I used to create a RouterDb database.

```
        public static RouterDb CreateRouterDb()
        {
            var routerDb = new RouterDb();
            using (var stream = File.OpenRead(@"Z:\OSM\planet\europe\italy-latest.osm.pbf"))
            {
                var source = new OsmSharp.Osm.PBF.Streams.PBFOsmStreamSource(stream);
                var progress = new OsmSharp.Osm.Streams.Filters.OsmStreamFilterProgress();
                progress.RegisterSource(source);
                routerDb.LoadOsmData(progress, Vehicle.BigTruck);
            }

            routerDb.AddContracted(Vehicle.BigTruck.Fastest());
            routerDb.AddContracted(Vehicle.BigTruck.Shortest());

            //routerDb.AddContracted(Vehicle.Car.Classifications());
            //routerDb.AddContracted(Vehicle.Pedestrian.Fastest());
            //routerDb.AddContracted(Vehicle.Bicycle.Fastest());
            //routerDb.AddContracted(Vehicle.Bicycle.Networks());

            using (var stream = File.OpenWrite(@"italy.routing"))
            {
                routerDb.Serialize(stream);
            }

            return routerDb;
        }
`

**Comment by pajaro5** [comment]
Hi @alecava58 

Look at this:
![image](https://cloud.githubusercontent.com/assets/5140280/16623865/2cf584b4-4365-11e6-81d3-ac6d72231b36.png)

I'm not able to find the RouterDB inside OsmSharp.Routing namespace,  even looking with the reflector object browser
![image](https://cloud.githubusercontent.com/assets/5140280/16624129/26bdb192-4366-11e6-8588-27377fc874e2.png)

What I'm missing?

Thanks for your time.
PR

**Comment by xivk** [comment]
What .NET version are you targetting? What package have you installed?

**Comment by pajaro5** [comment]
Thanks for your kindly attention @xivk,  @alecava58 

The problem its now resolved,  I deleted the OsmSharp.Routing dll and run from the nuget console this command:

`Install-Package OsmSharp.Routing -IncludePrerelease`

![image](https://cloud.githubusercontent.com/assets/5140280/16627744/3f504bd0-4374-11e6-9855-b7430333258f.png)

The difference was that at the first time I installed it using the Nuget Admin window instead the console.

Now I'm going to try to generate the  OsmSharp-routing file.

Thanks again.
